src/menu_parser.py

```diff
@@ -174,7 +174,7 @@ def get_menus(self, text, year, week_number):
             # remove allergnes
             for allergen in self.allergens:
                 # only replace "whole word matches" not followed by a hyphen (e.g. some dishes include "Senf-")
-                lines_weekdays[key] = re.sub(r"\b%s\b(?![\w-])" % allergen, "", lines_weekdays[key])
+                lines_weekdays[key] = re.sub(r"(\b%s\b(?![\w-])|\.\/\.)" % allergen, "", lines_weekdays[key])
```

Comment:
wouldn't it make things less complicated to add ./. to the list of allergens?

Reply:
Did not work for me. I think it is related to Regex checking for words.

Reply:
yeah, I think I think the word boundary \b is the issue. Still, I do not think that extending the regex is the right approach. Could you do something like

```
lines_weekdays[key] = lines_weekdays[key].replace("./.", "")
```

below the for loop?